content/en/docs/Reference/operator.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Operators"
+linkTitle: "Operators"
+weight: 3
+date: 2021-03-15
+---
+## What is an Operator?
+
+Operators make it easy to manage complex stateful applications on top of Kubernetes. By [Extendig the Kubernetes API with CustomResourceDefinitions](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/) it is possible to create what we call Operator Projects where you will be able to create and define your own API's to represent your solutions on the cluster and managed them by looking for the desired state. For a better understanding see [Operator Pattern][operator-pattern]
+
+## What is the difference between a Controller and an Operator
+
+See that following the [Operator's pattern][operator-pattern]; you will create [Controllers](https://kubernetes.io/docs/concepts/architecture/controller/) which provides a reconcile function responsible for synchronizing the resources until a desired state on the cluster. For a more practical explanation, see also [What’s in a controller?](https://book.kubebuilder.io/cronjob-tutorial/controller-overview.html#whats-in-a-controller)
+
+## What is Operator Lifecycle Manager?
+
+Operator Lifecycle Manager (OLM) helps users install, update, and manage the lifecycle of all Operators and their associated services running across their clusters. This project is a component of the [Operator Framework][operator-framework], an open-source toolkit to manage Kubernetes native applications, called Operators, in an effective, automated, and scalable way. Read more in the introduction [blog post](https://www.openshift.com/blog/introducing-the-operator-framework?extIdCarryOver=true&sc_cid=701f2000001Css5AAC).
+
+## How does OLM enable cluster admins and Developers?
```

Comment:
Super minor nit, can we normalize the capitalization of non-articles in this head description?

```suggestion
## How does OLM enable cluster admins and developers?
```

Or,

```suggestion
## How Does OLM Enable Cluster Admins and Developers?
```

Reply:
I have made all the changes that you have suggested. Thanks

Reply:
Thanks for all the suggestions. I have pushed all the changes.